fv3core/stencils/d_sw.py

```diff
@@ -93,8 +94,92 @@ def horizontal_relative_vorticity_from_winds(
         vorticity[0, 0, 0] = rarea * (vt - vt[0, 1, 0] - ut + ut[1, 0, 0])
 
 
+@gtscript.function
+def add_dw(w, dw, damp_w):
+    w = w + dw if damp_w > 1e-5 else w
+    return w
+
+
+@gtscript.function
+def ke_from_bwind(ke, ub, vb):
+    return 0.5 * (ke + ub * vb)
+
+
+@gtscript.function
+def adjust_w_and_qcon(w, delp, dw, damp_w, q_con):
+    w = w / delp
+    w = add_dw(w, dw, damp_w)
+    q_con = q_con / delp
+
+    return w, q_con
+
+
 @gtstencil()
-def not_inlineq_pressure(gx: sd, gy: sd, rarea: sd, fx: sd, fy: sd, pt: sd, delp: sd):
+def ke_horizontal_vorticity_w_qcon_adjust(
+    ke: FloatField,
+    u: FloatField,
+    v: FloatField,
+    ub: FloatField,
+    vb: FloatField,
+    ut: FloatField,
+    vt: FloatField,
+    dx: FloatField,
+    dy: FloatField,
+    rarea: FloatField,
+    vorticity: FloatField,
+    w: FloatField,
+    delp: FloatField,
+    dw: FloatField,
+    q_con: FloatField,
+    dt: float,
+    nested: bool,
+    damp_w: float,
+):
+    from __externals__ import (
+        i_end,
+        i_start,
+        j_end,
+        j_start,
+        local_ie,
+        local_is,
+        local_je,
+        local_js,
+    )
+
+    with computation(PARALLEL), interval(...):
+        ke = ke_from_bwind(ke, ub, vb)
+
+        with horizontal(region[i_start, j_start]):
+            if not nested:
+                ke = corners.corner_ke(ke, u, v, ut, vt, dt, 0, 0, -1, 1)
+        with horizontal(region[i_end + 1, j_start]):
+            if not nested:
+                ke = corners.corner_ke(ke, u, v, ut, vt, dt, -1, 0, 0, -1)
+        with horizontal(region[i_end + 1, j_end + 1]):
+            if not nested:
+                ke = corners.corner_ke(ke, u, v, ut, vt, dt, -1, -1, 0, 1)
+        with horizontal(region[i_start, j_end + 1]):
+            if not nested:
+                ke = corners.corner_ke(ke, u, v, ut, vt, dt, 0, -1, -1, -1)
+
+        vt = u * dx
+        ut = v * dy
+        vorticity[0, 0, 0] = rarea * (vt - vt[0, 1, 0] - ut + ut[1, 0, 0])
```

Comment:
This is from the stencil horizontal_relative_vorticity_from_winds. You should probably delete that stencil as it is no longer used, except that it is used by the VorticityVolumeMean test. SInce it has a docstring it would be nice to preserve that work. You could make that a gtscript function, include the docstring, and call it from here. For the test, you can either adapt it to create a stencil in the test class (such as for d2a2c here https://github.com/VulcanClimateModeling/fv3core/commit/95456e9d15190519356adaecbbc559bdbbbf4d78#diff-8e0363bcb510536bac792406fdf19fb833aa3782aaa0d026afac68f87e456ec9), or possibly just delete the test since this is really a tiny amount of code.

Reply:
I'm not sure how this is the case, but the VorticityVolumeMean test seems to be run every time I run the d_sw test case, and the test will complain that it can't find the `horizontal_relative_vorticity_from_winds` stencil if its removed.

Reply:
Are you specifying --which_modules=D_SW when running the tests? Either way the way to delete a test is to remove the translate class file and remove reference to it in tests/translate/__init__.py. Or you can make a stencil inside of the VorticityVolumeMean test class that alls your function

Reply:
I was specifying `--which_moudles=D_SW` during my tests, but the VoticityVolumeMean test was still running.  I ended up deleted the particular test and removing the reference in `tests/translate/init.py`.